doc/themes/nilearn/static/nature.css_t

```diff
@@ -1037,6 +1037,13 @@ table.field-list tr td.field-body p.first {
 table.field-list tr td.field-body p {
     margin-top: 0.5em;
 }
+
+/* Fix spacelessness between parameter name and description */
+dl.field-list span.classifier {
+    margin-left: 1ex;
+}
```

Comment:
Hum, it seems to me that this PR is mixing commits from your other PR and from the original one. It would be best to start from a clean PR, because it will be very hard to review.

Reply:
Indeed. I noticed it too late and tried to fix it but fell in a git-hole. I'll recreate the branch and open a new PR tomorrow.